instrumentation/opentelemetry-instrumentation-tornado/README.rst

```diff
@@ -0,0 +1,39 @@
+OpenTelemetry Tornado Instrumentation
+======================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-instrumentation-tornado.svg
+   :target: https://pypi.org/project/opentelemetry-instrumentation-tornado/
+
+This library builds on the OpenTelemetry WSGI middleware to track web requests
+in Tornado applications.
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-instrumentation-tornado
+
+Configuration
+-------------
+
+The following environment variables are supported as configuration options:
+
+- OTEL_TORNADO_EXCLUDED_URLS 
```

Comment:
the env variable should be `OTEL_PYTHON_` as per https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#language-specific-environment-variables